.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--
+Please use this issue tracker only for reporting bugs or feature requests.
+
+If you need support, please use:
+ 
+- our community forum
+  http://discourse.algolia.com
+  
+- StackOverflow with the `algolia` tag
+  https://stackoverflow.com/questions/tagged/algolia
+
+-->
+
+- Django version: 
+- Algolia Django integration version: 
+- Algolia Client Version: #.#.#
```

Comment:
This is not necessary, they should never include the client directly so just with the Django version we know what client is used underneath 😉

Reply:
I'm not very familiar with pip yet but typically in PHP, if you decide to include the symfony integration, it will also install the algolia client but it will decide what's the best version at that time (following the requirements). So usually you get the latest version, but if you could upgrade the integration without upgrading the client. Or we could release a few version of the client while not releasing any version of the intergation, then you're not up to date.
I don't know if I'm very clear or if that makes sense in python.

Reply:
Hey, I was not sure so I tested it:
`pip freeze`
> ```
> algoliasearch==1.12.0
> algoliasearch-django==1.5.2
> ```
`pip install algoliasearch==1.13.0 && pip freeze`
> ```Collecting algoliasearch==1.13.0
>   Downloading algoliasearch-1.13.0-py2.py3-none-any.whl
> Requirement already satisfied: requests[security]>=2.9.1 in /Users/pln/.virtualenvs/djangoEx/lib/python3.6/site-packages (from algoliasearch==1.13.0)
> ...
> Installing collected packages: algoliasearch
>   Found existing installation: algoliasearch 1.12.0
>     Uninstalling algoliasearch-1.12.0:
>       Successfully uninstalled algoliasearch-1.12.0
> Successfully installed algoliasearch-1.13.0
> You are using pip version 9.0.1, however version 9.0.3 is available.
> You should consider upgrading via the 'pip install --upgrade pip' command.
> 
> algoliasearch==1.13.0
> algoliasearch-django==1.5.2
> ```